### DISCUSSION TITLE
（灌水）有个问题...

### DISCUSSION THREAD

**Discussion by qmtqyh** [share121/inter-knot#2601] [其他]
有个问题，发R18或者R16图片会被删吗（好奇）
![C1BEDC40244178D1E1D3FAE369E71511](https://github.com/user-attachments/assets/33ee2e3f-98e7-4f51-8255-e0a23bfe0bf3)

**Comment by liusugong** [comment]
真理在于实践

**Comment by HifumiMizuhara** [comment]
网站定位 16+, 18-。
🟢：仅为让人联想起和性有关的东西，包括亲吻，拥抱，内衣的露出，能引起性行为的联想的表情等，胸部，臀部的强调；穿着极端面积狭小的衣物的人物描写等
❌：关联性的，或者表现性qi的结合，或者能够让人联想起性行为的。

**Reply by qmtqyh** [reply]
谢谢大佬qwq(
